cli/exp_rpty.go

```diff
@@ -149,6 +142,15 @@ func handleRPTY(inv *serpent.Invocation, client *codersdk.Client, args handleRPT
 		}()
 	}
 
+	// If a user does not specify a command, we'll assume they intend to open an
+	// interactive shell.
+	var backend string
+	if isOneShotCommand(args.Command) {
+		// If the user specified a command, we'll prefer to use the buffered method.
+		// The screen backend is not well suited for one-shot commands.
+		backend = "buffered"
+	}
```

Comment:
Suggestion: It'd be nice to see this as a CLI flag too, to (`auto`, `plain`, `screen`).

Side-note: I guess `screen` could even be called `multiplexer` to allow to swap out for e.g. tmux in the future.

Reply:
Yeah, I considered having it a CLI flag, but hesitated exposing this internal implementation detail. It does make sense, especially with your `vim` comment below.